IBAnimatable/TransitionAnimationType.swift

```diff
@@ -190,6 +190,32 @@ extension TransitionAnimationType {
         return .cross
       }
     }
+
+    var matchingGesture: InteractiveGestureType.GestureDirection {
+      switch self {
+      case .left: return .left
+      case .right: return .right
+      case .top: return .top
+      case .bottom: return .bottom
+      case .forward: return .open
+      case .backward: return .close
+      default:
+        fatalError("No matching GestureDirection for \(self)")
+      }
+    }
+
+    var opposingGesture: InteractiveGestureType.GestureDirection {
+      switch self {
+      case .left: return .right
+      case .right: return .left
+      case .top: return .bottom
+      case .bottom: return .top
+      case .forward: return .close
```

Comment:
Can you put your comment into the code as a `//blah blah` for future reference? That can help us remember what we did. It is very cool actually.